examples/aditof-demo/aditofdemoview.cpp

```diff
@@ -41,48 +41,17 @@
 
 namespace detail {
 std::string getKeyPressed(int key, bool &backspace) {
-    switch (key & 0xff) {
-    case 46:
-        return ".";
-    case 48:
-        return "0";
-    case 49:
-        return "1";
-    case 50:
-        return "2";
-    case 51:
-        return "3";
-    case 52:
-        return "4";
-    case 53:
-        return "5";
-    case 54:
-        return "6";
-    case 55:
-        return "7";
-    case 56:
-        return "8";
-    case 57:
-        return "9";
-    case 97:
-        return "a";
-    case 98:
-        return "b";
-    case 99:
-        return "c";
-    case 100:
-        return "d";
-    case 101:
-        return "e";
-    case 102:
-        return "f";
-    case 8:
-        backspace = true;
-        return "";
-
-    default:
+    key = key & 0xff;
+    if (key < 128) {
+        if (key == 8) {
+            backspace = true;
+            return "";
+        }
+        char c = key;
+        std::string val(&c);
```

Comment:
what if dot character is being entered? Will it work?

Reply:
I suppose it will. But please make sure to test if it works.

Reply:
With the latest update it works. It has been tested on linux, windows and dragonboard.